scripts/setup/macos-10.15/install_deps.sh

```diff
@@ -7,7 +7,7 @@ set -eux
 # Github promises weekly build image updates, so we can skip the update step and
 # worst case we should only be 1-2 weeks behind upstream brew.
 # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-software
-brew update
+#brew update
```

Comment:
Unfortunately, we might need to keep this :/

Even with the version pin, I think it has to be in the index?

Reply:
Based on [the documentation](https://docs.brew.sh/Taps#the-brew-tap-command), `brew tap` clones the repository, so a `brew update` should not be needed (at least for cbmc, not sure about other packages).